lib/web_ui/dev/test_platform.dart

```diff
@@ -60,12 +65,18 @@ class BrowserPlatform extends PlatformPlugin {
     );
   }
 
+  // Browsers supported by this platform.
+  final SupportedBrowsers supportedBrowsers = SupportedBrowsers();
+
   /// The test runner configuration.
   final Configuration _config;
 
   /// The underlying server.
   final shelf.Server _server;
 
+  // Name for the running browser. Not final on purpose should be changed later.
```

Comment:
> `Not final on purpose should be changed later.`

There seems to be a grammar issue in this sentence. Also what needs to be changed later? "Changed" as in "mutated"? :thinking:

Reply:
done.